server/models.py

```diff
@@ -586,6 +595,31 @@ def revision(self, user_ids):
                       .order_by(Backup.created.desc())
                       .first())
 
+    def scores(self, user_ids, only_published=True):
```

Comment:
Since active assignments don't display scores - is it worth it to add a flag that doesn't include scores? My guess is no - but it would reduce the number of queries we make when there are lots of active assignments.